Marlin/Marlin_main.cpp

```diff
@@ -2238,28 +2238,15 @@ static void clean_up_after_endstop_or_probe_move() {
    *
    * @return The raw Z position where the probe was triggered
    */
-  #define HAS_CALIBRATION_PROBE (ENABLED(DELTA_AUTO_CALIBRATION) && Z_PROBE_LOW_POINT < 0)
-  static float run_z_probe(
-    #if HAS_CALIBRATION_PROBE
-      const bool is_calibration
-    #endif
-  ) {
+  static float run_z_probe() {
 
     #if ENABLED(DEBUG_LEVELING_FEATURE)
       if (DEBUGGING(LEVELING)) DEBUG_POS(">>> run_z_probe", current_position);
     #endif
 
-    #if Z_PROBE_LOW_POINT < 0
-      // Stop the probe before it goes too low to prevent damage.
-      // If Z isn't known or this is a "calibration probe" then probe to -10mm.
-      #if !HAS_CALIBRATION_PROBE
-        constexpr bool is_calibration = false;
-      #endif
-      const float z_probe_low_point = !is_calibration && axis_known_position[Z_AXIS] ? -zprobe_zoffset + Z_PROBE_LOW_POINT : -10.0;
-    #else
```

Comment:
Z_PROBE_LOW_POINT testing was already done in SanityCheck.h

Reply:
Actually, this is all very conscientious. The sanity check just ensures that the value is set to zero (off) or negative (good), and throws an error if users set the value positive (wrong). The `#if` statement here could be `#if Z_PROBE_LOW_POINT` but it is redundant instead, just in case someone overrides the sanity check.

Reply:
Sanity check made sure it was negative. This checks whether it's on (and also negative, just in case).